src/main/cpp/chassis/states/MaintainHeading.cpp

```diff
@@ -15,6 +15,8 @@
 #include <cmath>
 #include <string>
 
+constexpr double PI = 3.14159265358979323846; // Define PI as a constant
```

Comment:
Why can't we use the standard math PI

Reply:
My computer wouldn't build with the math::PI for some reason. I had to define it, but I wonder if it was with the simulation build and I can remove it. I will try it and make the changes if it works

Reply:
To get to to not error, I had to change the it from "PI" to std::numbers::pi I don't know what changed, but something did. Especially since this was the only change in this file.

Reply:
std::numbers::pi is the new constant to use